### PR TITLE
Update knative install using Gloo

### PR DESCRIPTION
**Pull request by MIBc**
## Proposed Changes

- Update shell `glooctl install knative`, the previous one only install
knative-serving.
- Update outdated url.

### CODE REVIEW

File: docs/install/Knative-with-Gloo.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ Now you can deploy an app using your freshly installed Knative environment.
 ## Configuring DNS
 
 Knative dispatches to different services based on their hostname, so it greatly
-simplifies things to have DNS properly configured. For this, we must look up the
+simplify things to have DNS properly configured. For this, we must look up the
```
Comment (samodell) [inline docs/install/Knative-with-Gloo.md]: the previous wording was correct; please leave "simplifies"

Comment (MIBc) [reply docs/install/Knative-with-Gloo.md]: done

**Comment by MIBc** [comment]
@samodell Please review it

**Comment by MIBc** [comment]
@samodell Do you have any other suggestions？

**Pull request merged**